gnosis/safe-react
# Enable Walletconnect owners for Polygon and BSC

## What it solves
Resolves #2586

## How this PR fixes it
Enable Walletconnect in allowed wallets for BSC and Polygon

## How to test it
Connect using a wallet connect owner, for example using Metamask mobile app and see that you can normally operate with the safe
